src/client/components/pages/Schedule/blocks/SessionBlock.tsx

```diff
@@ -116,10 +116,14 @@ const SessionBlockWrapper = styled.div<SessionBlockWrapperProps>`
   border-left: 1px solid #fff;
   border-right: 1px solid #fff;
   border-bottom: 1px solid #fff;
-  min-width: 2.5em;
+  min-width: 3.5em;
   position: relative;
   opacity: ${({ isFaded }) => (isFaded ? '0.6' : '1')};
-
+ 
+  ::-webkit-scrollbar {
+    width: 0.2em;
+    height: 0.2em
+}
```

Comment:
The `-webkit-scrollbar` pseudo-selector isn't [supported on Firefox][1], so this won't be as helpful to the situation there. I did some searching and found [a different css-only solution on stack overflow][2] that uses the `:hover` pseudo-selector, leaving the `overflow-y` set to hidden when the mouse in elsewhere and `auto` when the mouse is over the block.* That seems to work consistently across Firefox/Edge on Windows and Firefox on Linux. 

In practice, the change looks like:

```diff
 const SessionBlockBodyWrapper = styled.div<SessionBlockBodyWrapperProps>`
-  overflow-y: ${({ isPopoverVisible }): string => (
-    isPopoverVisible
-      ? 'hidden'
-      : 'scroll'
-  )};
+  overflow-y: hidden;
   position: absolute;
   width: 100%;
   top: 2em;
   bottom: 0;
   border-top: 1px solid rgba(255,255,255,0.5);
+  &:hover {
+    overflow-y: ${({ isPopoverVisible }): string => (
+    isPopoverVisible
+      ? 'hidden'
+      : 'auto'
+  )};
+  }
 `;
```


I think it's OK to hold onto the change in `min-width` and the `-webkit-scrollbar` as it does improve the look overall, but I think that change will help us get more consistent behavior across the different browsers/OS combinations.

_* A user can tab through all of the course listings on the page, so it's OK to have this behavior require the mouse._

[1]: https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar
[2]: https://stackoverflow.com/a/63348534